### DISCUSSION TITLE
Does this project make use of any of these techniques?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/eepp#59] [Q&A] [answered]
- Build with sanitizers
- Run static analyzers
- Memory profiling

It's kind of an interview question. Feel free to dismiss it.

**Comment by ghost** [comment]
Oh sorry. I needed to ask some C++ projects to do a survey. I don't notice the repository. I have said that I will not bother you.

**Comment by SpartanJ** [comment, marked as answer]
I'm not bothered if you keep being respectful. This goes to [Discussions](https://github.com/SpartanJ/eepp/discussions) though, but no problem.

I'll answer you:

> Build with sanitizers

Just internally for testing (ThreadSanitizer, Memory and Address Sanitizers). eepp debug builds also have an internal memory sanitizer.

> Run static analyzers

Yes, mostly cppcheck and clang-analyzer.

> Memory profiling

Yes, valgrind.
